### DISCUSSION TITLE
chatglm3-6b在服务器上部署时遇到 'latin-1' codec can't encode characters的问题

### DISCUSSION THREAD

**Discussion by DerekAI** [zai-org/ChatGLM3#763] [Q&A] [answered]
在一台GPU服务器上部署ChatGLM3-6B，使用cli_demo.py测试时，出现如下错误
UnicodeEncodeError: 'latin-1' codec can't encode characters

请问是如何解决这个编码问题的？ latin-1转utf-8？

说明一下：服务器无root权限

谢谢

**Comment by DerekAI** [comment]
在服务器上vim查看源代码时
welcome_prompt = "__[0m~N__[0m~T ChatGLM3-6B __[0m~^~K[0m~L[0m~S[0m~E[0m~F~E__[0m~M[0m~O[0m~[[0m~L__[0m~][0m~Lclear [0m~E____[0m~][0m~N~F[0m~O[0m~Lstop [0m~H__[0m~K[0m~O"
起始的欢迎prompt也如上所示

在文件头部添加#coding=utf-8的方法也不奏效

**Reply by zRzRzRzRzRzRzR** [reply, marked as answer]
暂时没遇到相关错误，但是大概率是编码问题
